options/components/webTopContent/WebTopContent.jsx

```diff
@@ -0,0 +1,20 @@
+import WebGlobalNavigationBar from "./top-contents/WebGlobalNavigationBar";
+import WebKeywordSearchBox from "./top-contents/WebKeywordSearchBox";
+
+export default function WebTopContent({ urlNewList }) {
+  const selectValueTypes = [
+    { selectType: "키워드 검색" },
+    { selectType: "제목 검색" },
+    { selectType: "제목 + 키워드 검색" },
+  ];
```

Comment:
selectBox의 문구들을 배열로 관리하는건 좋은 생각 같습니다. 다만 배열 안에 왜 객체로 자료구조를 짜셨는지 의문이 들어 리뷰 남깁니다. 각 객체 안에 key값이 같다면 다음과 같이 작성하는게 더 직관적이고 사용하기 편할 것 같습니다.
```suggestion
  const selectValueTypes = ["키워드 검색", "제목 검색", "제목 + 키워드 검색"];
```

Reply:
해당 객체들을 꺼내서 쓸 때 어떤 값인지 key값을 통해 유추할 수 있게 끔 하고자 객체로 사용하였으나 말씀 주신대로 동일한 key값이다 보니 큰 의미가 없을 수 있겠다 판단이 들어 제시해주신 배열을 참고하여 변경하겠습니다.